apps/oip-web/src/app/globals.css

```
@@ -0,0 +1,71 @@
+  --oip-walnut: #3a2a1b;
+  --oip-heading: #1f1d1a;
+  --oip-body: #2a2a2a;
+  --oip-muted: #6f675c;
+  --oip-cream-muted: #bfb8ab;
+  --oip-rule: color-mix(in oklab, var(--oip-soil) 18%, transparent);
+  --oip-card: color-mix(in oklab, var(--oip-paper) 86%, white);
+  --oip-figure-ground: #fffaf0;
+  --oip-figure-caption: #3a2a1b;
+  --oip-portrait-ground: #1f1d1a;
+  --oip-client-logo-filter: grayscale(1) brightness(0) contrast(1.14);
+  color-scheme: light;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)